### ISSUE TITLE
Unable to install Graphon

### ISSUE THREAD

**Issue by mshaheer92** [vinivendra/homebrew-gryphon#3]
I'm getting the below error on running **brew install vinivendra/gryphon/gryphon**

==> Downloading https://rubygems.org/downloads/xcodeproj-1.16.0.gem
Already downloaded: /Users/shaheermalliyil/Library/Caches/Homebrew/downloads/9e7f02841e532e6e0113019acaa6459ebb34d50c4f4f856356ef4d38cfa5e2d4--xcodeproj-1.16.0.gem
==> Downloading https://github.com/vinivendra/Gryphon/archive/v0.18.1.tar.gz
Already downloaded: /Users/shaheermalliyil/Library/Caches/Homebrew/downloads/8b629ab4c4d811f72d9a6169fa3846e116eb5a4879c49493075429d3ad8dddb5--Gryphon-0.18.1.tar.gz
==> Installing gryphon from vinivendra/gryphon
==> Downloading https://rubygems.org/downloads/xcodeproj-1.16.0.gem
Already downloaded: /Users/shaheermalliyil/Library/Caches/Homebrew/downloads/9e7f02841e532e6e0113019acaa6459ebb34d50c4f4f856356ef4d38cfa5e2d4--xcodeproj-1.16.0.gem
==> gem install --install-dir /opt/homebrew/Cellar/gryphon/0.18.1/libexec/vendor /Users/shaheermalliyil/Library/Caches/Homebrew/downloads/9e7f02841e532e6e0113019acaa6459ebb34d50c4f4f856356ef4d38cfa5e2d4--xco
==> swift build --disable-sandbox
Last 15 lines from /Users/shaheermalliyil/Library/Logs/Homebrew/gryphon/02.swift:
             ^
/private/tmp/gryphon-20220330-41376-ivxr5l/Gryphon-0.18.1/Sources/GryphonLib/GryphonSwiftLibrary.swift:590:14: error: type 'MutableList<Element>' does not conform to protocol 'MutableCollection'
public class MutableList<Element>: List<Element>,
             ^
/private/tmp/gryphon-20220330-41376-ivxr5l/Gryphon-0.18.1/Sources/GryphonLib/GryphonSwiftLibrary.swift:590:14: error: unavailable subscript 'subscript(_:)' was used to satisfy a requirement of protocol 'MutableCollection'
public class MutableList<Element>: List<Element>,
             ^
Swift.MutableCollection:8:12: note: 'subscript(_:)' declared here
    public subscript(bounds: Range<Self.Index>) -> Self.SubSequence { get set }
           ^
Swift.MutableCollection:6:14: note: requirement 'subscript(_:)' declared here
    override subscript(bounds: Range<Self.Index>) -> Self.SubSequence { get set }
             ^
[137/139] Compiling GryphonLib TranspilationPass.swift
[138/139] Compiling GryphonLib Utilities.swift

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/vinivendra/homebrew-gryphon/issues

**Comment by nesalexy** [comment]
Have the same error on apple M1

```
Last 15 lines from /Users/nesalexy/Library/Logs/Homebrew/gryphon/02.swift:
    override subscript(bounds: Range<Self.Index>) -> Self.SubSequence { get set }
             ^
/private/tmp/gryphon-20220408-96403-1onb45o/Sources/GryphonLib/GryphonSwiftLibrary.swift:590:14: error: type 'MutableList<Element>' does not conform to protocol 'MutableCollection'
public class MutableList<Element>: List<Element>,
             ^
/private/tmp/gryphon-20220408-96403-1onb45o/Sources/GryphonLib/GryphonSwiftLibrary.swift:590:14: error: unavailable subscript 'subscript(_:)' was used to satisfy a requirement of protocol 'MutableCollection'
public class MutableList<Element>: List<Element>,
             ^
Swift.MutableCollection:8:12: note: 'subscript(_:)' declared here
    public subscript(bounds: Range<Self.Index>) -> Self.SubSequence { get set }
           ^
Swift.MutableCollection:6:14: note: requirement 'subscript(_:)' declared here
    override subscript(bounds: Range<Self.Index>) -> Self.SubSequence { get set }
             ^
[138/139] Compiling GryphonLib Utilities.swift

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/vinivendra/homebrew-gryphon/issues

```

**Comment by maxhuckers** [comment]
<img width="1286" alt="Screenshot 2022-04-09 at 19 42 15" src="https://user-images.githubusercontent.com/59177974/162587547-9c54f572-f638-404b-9bce-349d1af1a484.png">

I also keep encountering this issue, tried all three installation methods

**Comment by vinivendra** [comment]
Hey folks, thanks for the error report. That's weird, I thought I solved this last year, I even said I did on the [Swift forums](https://forums.swift.org/t/protocol-conformance-stopped-working-in-xcode-13/53345/2), but I can't find the bugfix anymore. I'll take a look at it asap.

**Comment by vinivendra** [comment]
OK, I figured it out. It's not the same bug at all, though it's similar, which made it kind of easy to solve.

The new problem is, you probably found this bug because you're running Xcode 13.3, which uses Swift 5.6, but Swift 5.6 isn't actually supported yet - and implementing that support might take a while.

If you try to use Swift 5.5 (e.g. by running `xcode-select` with Xcode 13.2.1 or older) it'll probably work. Otherwise, I hope to have support for Swift 5.6 up in a few days, if everything works out.

**Comment by vinivendra** [comment]
Fixed in Gryphon version `0.19` with support for Swift 5.6.
